### PR TITLE
Core: Move several files into the correct VS directory filter.

### PR DESCRIPTION
**Pull request by lioncash**

### CODE REVIEW

File: Source/Core/Core/Core.vcxproj.filters [hunk 1]
```diff
@@ -634,9 +634,6 @@
     <ClCompile Include="PowerPC\SignatureDB.cpp">
       <Filter>PowerPC</Filter>
     </ClCompile>
-    <ClCompile Include="PowerPC\Jit64Common\Jit64AsmCommon.cpp">
-      <Filter>PowerPC\Jit64Common</Filter>
-    </ClCompile>
```
Comment (magumagu) [inline Source/Core/Core/Core.vcxproj.filters]: Not sure we want to put Jit64AsmCommon.cpp into Jit64 category; there's probably going to be more files which move from JitCommon to Jit64Common in the near future.

The other changes look fine.

Comment (lioncash) [inline Source/Core/Core/Core.vcxproj.filters]: Ah true, good catch. Will fix it when I get home.

**Comment by lioncash** [comment]
Should be good now

File: Source/Core/Core/Core.vcxproj.filters [hunk 2]
```diff
@@ -136,6 +136,9 @@
     <Filter Include="PowerPC\JitILCommon">
       <UniqueIdentifier>{827afa93-1a80-4835-93ae-b5516d95867f}</UniqueIdentifier>
     </Filter>
+    <Filter Include="PowerPC\Jit64Common">
+      <UniqueIdentifier>{81956f71-d9fe-454f-96a6-855195d611c4}</UniqueIdentifier>
```
Comment (skidau) [inline Source/Core/Core/Core.vcxproj.filters]: Just a question (not a review comment): How are Unique ID's generated?

Comment (lioncash) [inline Source/Core/Core/Core.vcxproj.filters]: I don't know, Visual Studio automatically generates them when you create a filter through the UI.

**Pull request merged**